docroot/profiles/favrskov/modules/custom/favrskov_helper/favrskov_helper.install

```diff
@@ -1002,3 +1002,11 @@ function favrskov_helper_update_7054() {
 function favrskov_helper_update_7055() {
   features_revert_module('media_imagestyles');
 }
+
```

Comment:
@mnakov and update hook like this one is only ok, if we are 100% sure that we have the latest feature exported from live before the deploy

Reply:
@lachezarvalchev, I confirm that the first thing that I did was to export the feature from live.